### PR TITLE
documentation: Add converted from pptx overview to documentation/

### PR DESCRIPTION
**Pull request by beorn-**
This provides a few examples, and hints about how to use hwbench.

### CODE REVIEW

File: documentation/overview.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+- __monitor__ : enable system monitoring via BMC \(redfish\)\, system \(turbostat\, sysfs\, …\)
+
+    - __Objects__:
+        - _Thermal_
+        - _Fan_
+        - _PowerConsumption_
+        - _PowerSupplies_
+        - _Frequencies_
+
+- __Support__: Generic redfish \+ OEM \(HPE / DELL\)
```
Comment (anisse) [inline documentation/overview.md:39]: Wrong level of indentation, it should under monitor, or have no bullet.

(and should say: supports)

File: documentation/overview.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
+HWbench
+
+Criteo Hardware Team
```
Comment (anisse) [inline documentation/overview.md:3]: I know it was auto-genarated, but the first two lines seem out of place.

File: documentation/overview.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+- __engine\_module__ : what module from selected  _engine_
+
+- __engine\_module\_parameter__ : a list of  _engine\_module_  tests to execute
+
+# Execution (example with AMD 8434P)
+
+* System info
+* Performance
+* Monitoring
```
Comment (anisse) [inline documentation/overview.md:80]: These three seem out of place. The goal was to say that the json output contains all of this.

File: documentation/overview.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+![](img/Hwbench-intro5.png)
+
+
+# Render (example with AMD 8434P)
+
+Compare & graph several traces file
+
+![](img/Hwbench-intro10.png)
+
+__The following graphs are:__
```
Comment (anisse) [inline documentation/overview.md:97]: You removed the Disclaimer image, but it is still in the PR (intro11.png)

File: documentation/overview.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+
+# What's next ?
+
+__In random order:__
+
+- Add more metrics: PDU & other power sensors\, thermal sensors
+- Add more engines: storage\, applicative sample loads
+- Add parallelism to combine several engines: i\.e\.:  _stress\-ng_  \+  _fio_
+- Add more graph types?
+- Publish as opensource project
```
Comment (anisse) [inline documentation/overview.md:155]: This one is done :smile:  !

File: documentation/overview.md (hunk 4 above)
Comment (beorn-) [reply documentation/overview.md:97]: removed

File: documentation/overview.md (hunk 1 above)
Comment (anisse) [reply documentation/overview.md:39]: This monitor bullet point is the only missing change on my side.

**Review by anisse — APPROVED** [review]

**Review by ErwanAliasr1 — APPROVED** [review]

**Pull request merged**